docs/xocto/localtime.md

```diff
@@ -19,6 +19,69 @@ from xocto import localtime
 
 See [xocto.localtime](https://github.com/octoenergy/xocto/blob/master/xocto/localtime.py) for more details, including examples and in depth technical details.
 
+## Localising datetimes between different timezones
+
+`localtime` can be used to change the timezone of a datetime e.g.
+
+```python
+london_tz = zoneinfo.ZoneInfo("Europe/London")
+dt = datetime.datetime(2024, 1, 1, tzinfo=london_tz)
+
+sydney_tz = zoneinfo.ZoneInfo("Australia/Sydney")
+localtime.as_localtime(dt, tz=sydney_tz)
+=> datetime.datetime(2024, 1, 1, 11, 0, tzinfo=zoneinfo.ZoneInfo(key='Australia/Sydney'))
+
+localtime.as_utc(dt)
+=> datetime.datetime(2024, 1, 1, 0, 0, tzinfo=zoneinfo.ZoneInfo(key='UTC'))
+```
+
+Behaviour around Daylight Savings Time (DST) boundaries - where
+clocks go forward or back - can be surprising e.g.
+
+```python
+# Clocks in GBR went forward on this day at 1AM => 2AM;
+# only one hour elapsed for the rest of the world.
+localtime.as_utc(datetime(2020, 3, 29, hour=2, tzinfo=london_tz)) -
+localtime.as_utc(datetime(2020, 3, 29, hour=0, tzinfo=london_tz))
+=> timedelta(seconds=3600)
+```
+
+You should consider the implications of DST when localizing to a
+different timezone - especially when working with ranges.
+
+### Non-existent datetimes
+
+Python datetimes can be constructed for times that don't
+actually exist in certain timezones, due to DST changes e.g.
+
+```python
+# Doesn't exist in GBR as at this time the clocks went
+# forwards one hour - to 2AM.
+datetime(2020, 3, 29, hours=1, tzinfo=london_tz)
+
+# Python datetime operations do not consider DST when
+# doing calculations on datetimes.
+datetime(2020, 3, 29, hour=2, tzinfo=london_tz) -
+datetime(2020, 3, 29, hour=1, tzinfo=london_tz)
+=> datetime.timedelta(seconds=3600)  # <== WRONG (should be 0)
+```
```

Comment:
This section is confusing, it's not obvious why it's wrong

Reply:
I've elaborated the comment a bit; does that help?

Maybe this example just isn't a good fit here. I think it's important knowledge, so I'd like to put it somewhere.